__tests__/app.test.js

```diff
@@ -117,3 +113,54 @@ describe("5. GET /api/users", () => {
       });
   });
 });
+
+describe("6. PATCH /api/articles/:article_id", () => {
+  test('should return the response(an object) that has the ', () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({inc_votes:2})
+      .expect(200)
+      .then((res) =>{
+        res.body.vote
+        expect(typeof res.body).toBe("object");
+        expect( res.body.article.votes).toBe(2);
+
+
+      });});
+
+      
+
+  test('based on what number it is given, the votes key should be updated by substracting that number from the  value', () => {
+    return request(app)
+    .patch("/api/articles/3")
+    .send({ inc_votes:-5})
+    .expect((200))
+    .then((res)=>{
+      res.body.votes
+      expect(res.body.article.votes).toBe(-5);
+    })
+  });
+  
+  test("based on what number it is given, the votes key should be updated by substracting that number from the  value", () => {
+    return request(app)
+      .patch("/api/articles/1000")
+      .send({ inc_votes: -5 })
+      .expect(404)
+      .then((res) => {
+          const message = res.body.error;
+          expect(message).toBe("No article was found with id: 1000");      });
+
+  });
+   test("based on what number it is given, the votes key should be updated by substracting that number from the  value", () => {
+     return request(app)
+       .patch("/api/articles/banana")
+       .send({ inc_votes: -5 })
+       .expect(404)
```

Comment:
This should be an error 400 (not a 404), because 'banana' is not a valid article id, article id's should be numbers.